doc/releasenotes/14_0_0_release_notes.md

```diff
@@ -0,0 +1,650 @@
+# Release of Vitess v14.0.0-RC1
```

Comment:
I think we only did this part for GA in the past? https://github.com/vitessio/vitess/commits/main/doc/releasenotes/13_0_0_release_notes.md

I'm not sure though. And also don't feel too strongly about whether or not the past should guide the present/future. :-) 

If we leave this in place then we'll just of course have to be sure to update/replace it for the GA release.

Reply:
Right, this will be replaced during GA